### PR TITLE
Add web UI controls to filter only tagged or all builds

### PR DESCRIPTION
**Pull request by okurz**
Using a new query parameter 'only_tagged=[0|1]' the list can be filtered.

Example screenshot:
openqa_limit_builds_current_bold_and_only_tagged

Related progress issue: https://progress.opensuse.org/issues/11052

Obsoletes #804

### CODE REVIEW

File: lib/OpenQA/WebAPI/Controller/Main.pm [hunk 1]
```diff
@@ -159,6 +160,14 @@ sub group_overview {
             $res->{$build}->{tag} = {type => $tag[1], description => $tag[2]};
         }
     }
+    if ($only_tagged) {
+        for my $build (keys %$res) {
+            print "\$build: $build\n";
```
Comment (aaannz) [inline lib/OpenQA/WebAPI/Controller/Main.pm]: This is debug print only, right?

**Comment by okurz** [comment]
@coolo tell me if you want to use this current version or wait for me learn angularjs :-)

File: lib/OpenQA/WebAPI/Controller/Main.pm (hunk 1 above)
Comment (okurz) [inline lib/OpenQA/WebAPI/Controller/Main.pm]: yes, fixed, thx.

**Comment by coolo** [comment]
but won't users be confused if they click 'show 400 builds' and they only get 50? :)

**Comment by okurz** [comment]
:-D yes, they wil be

**Comment by okurz** [comment]
updated:
* rebased on current #804 and adapted to also show current selection in list in bold, non-link

**Pull request merged**